packages/Telephony/Actions/BatchDTMF.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Runtime.CompilerServices;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+
+using AdaptiveExpressions.Properties;
+using Microsoft.Bot.Builder.Dialogs;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Components.Telephony.Actions
+{
+    /// <summary>
+    /// Aggregates input until it matches a regex pattern and then stores the result in an output property
+    /// </summary>
+    public class BatchDTMF : Dialog
```

Comment:
How do we combine voice and DTMF input in the prompt? ("Please say or type your credit card number"). Will this just work?

Reply:
Yes, as it is, it will just work.